src/components/with-error-boundary/error-boundary.tsx

```diff
@@ -0,0 +1,38 @@
+import { Component, ErrorInfo, ReactNode } from 'react';
+
+interface Props {
+  placeholder?: ReactNode;
+}
+
+interface State {
+  hasError: boolean;
+}
+
+class ErrorBoundary extends Component<Props, State> {
+  // eslint-disable-next-line react/state-in-constructor
```

Comment:
Skriv gjerne en fullstendig constructor som definerer state for å bli kvitt den.

https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/state-in-constructor.md